### PR TITLE
Update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by bkamins**

### CODE REVIEW

**Comment by bkamins** [comment]
@KristofferC - is there a better recommendation? `Pkg.develop` does not necessarily give master, but I thought that it is better than recommending using `add DataFrames#master`. What do you think?

**Comment by KristofferC** [comment]
I think `Pkg.add(name="DataFrames", rev="master")` is the best because it can easily be copy pasted.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ Thanks for taking the plunge!
   or related issues for context, before opening a new issue
 * Including minimal examples is greatly appreciated
 * If it's a bug, or unexpected behaviour, reproducing on the latest development version
-  (`Pkg.checkout("DataFrames")`) is a good gut check and can streamline the process,
+  (`Pkg.develop("DataFrames")`) is a good gut check and can streamline the process,
```
Comment (bkamins) [inline CONTRIBUTING.md:14]:
```suggestion
  (`Pkg.add(name="DataFrames", rev="master")`) is a good gut check and can streamline the process,
```

**Comment by bkamins** [comment]
@nalimilan - OK to merge?

**Review by nalimilan — APPROVED** [review]

**Pull request merged**